dist/aframe-master.js

```diff
@@ -23971,8 +23971,13 @@ module.exports = anime;
 			/* Stub */
 		};
 	}
-
-	Object.assign(WebVRManager.prototype, EventDispatcher.prototype);
+                                        
+	Object.assign(WebVRManager.prototype,{
+        addEventListener: EventDispatcher.prototype['addEventListener'],
+        hasEventListener: EventDispatcher.prototype['hasEventListener'],
+        removeEventListener: EventDispatcher.prototype['removeEventListener'],
+        dispatchEvent: EventDispatcher.prototype['dispatchEvent']
+    });                          
```

Comment:
EventDispatcher indeed changed to a class in three r128 so really long time ago
https://github.com/mrdoob/three.js/commit/669e43a51e2ae929dfa95c67ac5ee815aea9f45d
Good find for this one, because `Object.assign(WebVRManager.prototype, EventDispatcher.prototype);` just fails silently, this is weird, it indeed doesn't copy any of the methods to `WebVRManager`. But your workaround actually copy the methods.

Reply:
Ah you said it in the linked issue, methods in a class are not enumerable, gotcha, that's explain it.